tools/integrations/manage.sh

```diff
@@ -117,6 +117,13 @@ cmd_publish() {
     docker push "$versioned_image"
     docker push "$latest_image"
   fi
+  
+  # Checking it the image was successfully registered on DockerHub
+  TAG_URL="https://hub.docker.com/v2/repositories/${image_name}/tags/${image_version}"
+  DOCKERHUB_RESPONSE_CODE=$(curl --silent --output /dev/null --write-out "%{http_code}" ${TAG_URL})
+  if [[ "${DOCKERHUB_RESPONSE_CODE}" == "404" ]]; then
```

Comment:
some questions: 
1.is this API immediately consistent, or eventually consistent? if the latter,we may start failing a lot of builds for transient reasons
2. what is the expectation from an engineer once they run across this error? are they expected to republish? bump the version again? If their build fails, what is their actionable next step?

Reply:
1. I don't know. As of today the connector I published that led to a 404 on the dockerhub still remains 404.  I could add a `sleep` for 5 seconds in case there's a small delay before the tag is available on the API after the push.
2. Bumping again is the suggested action. Republishing won't work because the image is registered but not discoverable with the API.

Reply:
could you test #1 please? i just want to make sure this doesn't suddenly start introducing a ton of failures for a case which is 1% of the time. 

if we want to move forward, then for 2 could you add that in the log message? (the action the dev needs to take)

Reply:
I tested to run a publish workflow on `destination-e2e` which latest version was not published. It worked. I also tried with the 404 URL to test the behavior in case of bad API response, it also worked as expected: the job failed and the message I set is echoed.